extensions/ql-vscode/src/model-editor/methods-usage/methods-usage-data-provider.ts

```diff
@@ -146,7 +146,14 @@ export class MethodsUsageDataProvider
   getChildren(item?: MethodsUsageTreeViewItem): MethodsUsageTreeViewItem[] {
     if (item === undefined) {
       if (this.hideModeledMethods) {
-        return this.sortedTreeItems.filter((api) => !api.method.supported);
+        return this.sortedTreeItems.filter((api) => {
```

Comment:
I always forget before opening the initial PR, but I can add some tests that cover this behaviour change.